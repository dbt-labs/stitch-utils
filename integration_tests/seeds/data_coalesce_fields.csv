id,numeral,numeral__st,numeral__bl,version__string,version__double,properties__numeral,properties__numeral__st,properties__numeral__bl,__red_herring
1,1,,,pi,,1,,,not
2,,one,,,3.14,,one,,this
3,,,TRUE,,,,,TRUE,one