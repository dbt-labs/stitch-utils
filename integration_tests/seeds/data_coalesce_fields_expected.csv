id,numeral,version,properties__numeral,__red_herring
1,"1","pi","1","not"
2,"one","3.14","one","this"
3,"true",,"true","one"
